source/Pacioli/Pacioli.Tests/Tests/JournalTests.cs

```diff
@@ -14,8 +18,43 @@ public void JournalDoesNotAcceptUnbalancedEntries(decimal debit, decimal credit)
             journalEntry.Debits.Add(new JournalEntryItem { Amount = debit });
             journalEntry.Credits.Add(new JournalEntryItem { Amount = credit });
             bool postSuccessful = journal.PostEntry(journalEntry);
-            
+
             Assert.False(postSuccessful);
         }
+
+        [Theory, ClassData(typeof(JournalEntryHasToHaveCreditAndDebit_TestData))]
+        public void JournalEntry_Throws_Exception_On_Empty_Credit_Or_Debit(List<JournalEntryItem> debits, List<JournalEntryItem> credits)
+        {
+            Assert.Throws<ArgumentException>(() => new JournalEntry(debits: debits, credits: credits));
```

Comment:
I am not sure this follows the Arrange Act Assert pattern. Perhaps you could do something like
```CSharp
var action = new Action(() => new JournalEntry(debits: debits, credits: credits));

Assert.Throws<ArgumentException>(action);
```

Reply:
Ok done, shortened version though.